assets/css/properties_panel/_vehicle_properties_panel.scss

```diff
@@ -70,6 +70,25 @@
   }
 }
 
+.m-vehicle-properties-panel__map-open-link {
+  position: absolute;
+  // Above leaflet-panes
+  z-index: 401;
+  top: 10px;
+  // Adjacent to controls - control width is 40px
+  right: calc(env(safe-area-inset-right) + 40px + 0.5rem);
+  text-decoration: none;
+  background-color: $white;
+  padding: 0.5rem;
+  border-radius: 6px;
+  color: $color-gray-700 !important;
+  font-weight: 500;
```

Comment:
**suggestion:** we now have a standard set of font weights defined in `_skate_ui.scss`. I know 500 isn't one of them, but it might be worth asking design if they want to change this to either 400 or 600 to be more standardized - I ran into something similar with the search pane and design decided to change the weight in question to one of the standard ones.

Reply:
Discussed with design and updated to use semi-bold